oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxPowerSource.java

```diff
@@ -73,13 +79,20 @@ public LinuxPowerSource(String newName, double newRemainingCapacity, double newT
             if (psInfo.isEmpty()) {
                 continue;
             }
+            
+            LinuxPowerSource pSource = new LinuxPowerSource();
+            
             // Initialize defaults
             boolean isPresent = false;
             boolean isCharging = false;
-            String name = "Unknown";
+            String name = "unknown";

```

Comment:
Earlier you changed a lowercase unknown to an uppercase Unknown, and here you do the opposite.  No big deal which one you use, but should try to be consistent! :)